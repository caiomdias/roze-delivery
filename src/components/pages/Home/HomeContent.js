import React from 'react';
import Search from '../../common/search/Search'

const HomeContent = () => (
	<div className="home-content">
		<div className="container">
			<div className="content-wrapper">
				<h1>
					<strong>Bebidas geladas </strong> a <strong>preço </strong>de mercado na sua casa <strong>agora</strong>
				</h1>
				<Search
					type="address"
					placeholder="Insera seu endereço para ver preço"
				/>
			</div>
		</div>
	</div>
);

export default HomeContent;
