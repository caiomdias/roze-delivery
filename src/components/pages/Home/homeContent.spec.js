import React from 'react';
import renderer from 'react-test-renderer';
import HomeContent from './HomeContent';

jest.mock('react-router-dom', () => ({
    useLocation: jest.fn().mockReturnValue({
      pathname: '/',
      search: '',
      hash: '',
      state: null,
      key: '5nvxpbdafa',
	}),
	useHistory: jest.fn().mockReturnValue({
		push: jest.fn().mockReturnValue()
	})
}));

describe("Home Content Component", () => {
	it('render correctly Home Content component', () => {  
		const homeContentComponent = renderer.create(<HomeContent />);
		expect(homeContentComponent).toMatchSnapshot();
	});
});