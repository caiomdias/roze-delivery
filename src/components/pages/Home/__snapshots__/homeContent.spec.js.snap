// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Content Component render correctly Home Content component 1`] = `
<div
  className="home-content"
>
  <div
    className="container"
  >
    <div
      className="content-wrapper"
    >
      <h1>
        <strong>
          Bebidas geladas 
        </strong>
         a 
        <strong>
          preço 
        </strong>
        de mercado na sua casa 
        <strong>
          agora
        </strong>
      </h1>
      <div
        className="search"
      >
        <div
          className="search__input"
        >
          <div
            className="icon"
          />
          <input
            className="input"
            onChange={[Function]}
            placeholder="Insera seu endereço para ver preço"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
